com.microsoft.mrtk.input/Visualizers/Visuals/Models/LICENSE.md

```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2019 Amazon
```

Comment:
Amazon????!

Reply:
Yeah, it was all pulled from this project: https://github.com/immersive-web/webxr-input-profiles/tree/main/packages/assets

I remember discussing with the broader team and there being no issues with it @keveleigh